packages/jss-preset-default/package.json

```diff
@@ -6,6 +6,7 @@
   "homepage": "https://cssinjs.org/jss-preset-default",
   "main": "dist/jss-preset-default.cjs.js",
   "module": "dist/jss-preset-default.esm.js",
+  "unpkg": "dist/jss-preset-default.bundle.js",
```

Comment:
does bundle suffix sound logical? I thought esm, cjs etc are all bundle types, what would be a corresponding more specific thing?

Reply:
I use the `bundle` suffix in Material Motion and Indefinite Observable, because it doesn't have any imports and can be used directly in the browser.  I followed the same convention here.

Reply:
FWIW, those libraries should be going on Google Hosted Libraries with the `.bundle.js` suffix this week.

Reply:
so its convention over there? ok then